src/parallel_parameter_search/walk_engine_optimization.py

```diff
@@ -218,21 +177,21 @@ def fix(name, value):
 class WolfgangWalkEngine(AbstractWalkEngine):
     def __init__(self, namespace, gui, walk_as_node, sim_type='pybullet', repetitions=1, multi_objective=False):
         super(WolfgangWalkEngine, self).__init__(namespace, gui, 'wolfgang', walk_as_node, sim_type,
-                                                 start_speeds=[0.2, 0.05, 0.25], repetitions=repetitions,
+                                                 start_speeds=[0.05, 0.025, 0.1], repetitions=repetitions,
                                                  multi_objective=multi_objective)
         self.reset_height_offset = 0.012
 
     def suggest_walk_params(self, trial):
-        self._suggest_walk_params(trial, trunk_height=(0.3, 0.45), foot_distance=(0.15, 0.25), foot_rise=(0.05, 0.2),
+        self._suggest_walk_params(trial, trunk_height=(0.39, 0.43), foot_distance=(0.15, 0.25), foot_rise=(0.05, 0.15),
```

Comment:
Is a trunk height of 0.43 even achievable with our robot? I thought, our base link is approx 04m high with fully extended legs.

Reply:
It is actually achievable, the base is something like 0.43 when fully extended. 0.43 itself does not work well since it is close to the kinematic singularity, but that's something the algorithm finds out by itself.